tidy3d/plugins/waveguide/rectangular_dielectric.py

```diff
@@ -68,18 +71,18 @@ class RectangularDielectric(Tidy3dBaseModel):
         discriminator=TYPE_TAG_STR,
     )
 
-    clad_medium: MediumType = pydantic.Field(
+    clad_medium: Union[AnnotatedMedium, List[AnnotatedMedium]] = pydantic.Field(
```

Comment:
The convention we've been using is to use tuples so that things are enforced to be immutable. but I dont know if we need to be as strict about this in plugins (and maybe generally it would have been ok to keep as lists as long as we don't internally mutate things)

Reply:
Tuple is fine. I had it as tuple before and changed to list because I was having trouble with the `AnnotatedMedium` and wanted to check. Then I forgot to change back.